src/graphql/GlobalSearch/search-by-id.gql

```diff
@@ -1,6 +1,7 @@
 query GlobalSearchByID($id: uuid!) {
-  flow(where: { id: { _eq: $id } }) {
+  flow(where:{ _or: [{ id: { _eq: $id } }, {flow_group_id: {_eq: $id}}]}) {
     id: flow_group_id
+    flow_id: id
```

Comment:
Haha these two lines really confused me for a minute, since both the left and right hand side are valid query attributes but 👍 I figured it out